base_layer/wallet_ffi/src/lib.rs

```diff
@@ -235,6 +236,9 @@ pub struct EmojiSet(Vec<ByteVector>);
 #[derive(Debug, PartialEq)]
 pub struct TariSeedWords(Vec<String>);
 
+#[derive(Debug, PartialEq)]
+pub struct TariMnemonicWordList(Vec<String>);
```

Comment:
Please just reuse TariSeedWords

Reply:
That way you don't need all the accessor and destroy functions to be repeated.

Reply:
@philipr-za `TariSeedWords` is to get the words in and out of the wallet. `TariMnemonicWordList` looks like it to refer to a dictionary all of words in the selected language. The former allows words to be added/removed, the latter does not. This may cause confusion if combined, so whilst the types are equivalent the usage is not.